example/05_conv2d_fwd/README.md

```diff
@@ -1,57 +0,0 @@
-# Instructions for ```conv2d_fwd_xdl``` Example
```

Comment:
Is this file deleted by accident?

So are some other files in example folder

Reply:
No this is deliberately since this file is now redundant. There is example 9 convnd_fwd which provides more functionality https://github.com/ROCmSoftwarePlatform/composable_kernel/blob/4248ed7ede401859868447462ebf71903bb33e03/example/09_convnd_fwd/convnd_fwd_xdl.cpp. Similar situation is for example 8.

Reply:
I thought that there is no reason to keep examples which are doing pretty the same.